test/pkg/environment/common/expectations.go

```diff
@@ -580,31 +584,31 @@ func (env *Environment) EventuallyExpectAvgUtilization(resource v1.ResourceName,
 	}, 10*time.Minute).Should(Succeed())
 }
 
-func (env *Environment) ExpectDaemonSetEnvironmentVariableUpdated(obj client.ObjectKey, name, value string) {
-	env.ExpectDaemonSetEnvironmentVariableUpdatedWithOffset(1, obj, name, value)
-}
-
-func (env *Environment) ExpectDaemonSetEnvironmentVariableUpdatedWithOffset(offset int, obj client.ObjectKey, name, value string) {
+func (env *Environment) ExpectDaemonSetEnvironmentVariableUpdated(obj client.ObjectKey, name, value string, containers ...string) {
+	GinkgoHelper()
 	ds := &appsv1.DaemonSet{}
-	ExpectWithOffset(offset+1, env.Client.Get(env.Context, obj, ds)).To(Succeed())
-	ExpectWithOffset(offset+1, len(ds.Spec.Template.Spec.Containers)).To(BeNumerically("==", 1))
+	ExpectWithOffset(1, env.Client.Get(env.Context, obj, ds)).To(Succeed())
```

Comment:
```suggestion
	Expect(env.Client.Get(env.Context, obj, ds)).To(Succeed())
```

You can just use `Expect` if you are using `GinkgoHelper`